pandas/core/sparse/frame.py

```diff
@@ -282,6 +282,21 @@ def _unpickle_sparse_frame_compat(self, state):
 
     @Appender(SparseFrameAccessor.to_dense.__doc__)
     def to_dense(self):
+        """
+        .. deprecated:: 0.25.0
+            Use Dataframe.sparse.to_dense() instead
+        """
+
+        warning_message = """\
+SparseDataFrame.to_dense is deprecated and will be removed in a future version
+
+            Use Dataframe.sparse.to_dense() instead
+
+            >>> df = pd.DataFrame({"A": pd.SparseArray([0, 1, 0])})
+            >>> df.sparse.to_dense()
```

Comment:
I was just thinking, maybe we don't need to deprecate this one, as it is on the SparseDataFrame class, which is already deprecated by itself?

Reply:
Good point. Agreed. We probably don't need this.